robot_tests/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3
```

Comment:
Can you please rename Dockerfile name as test.Dockerfile?
This would be helpful as we will have many dockerfiles in future for different tasks if needed.

Reply:
Also You can move the dockerfile inside openshift folder as all dockerfiles will be well organized at same place

Reply:
Done!